styles/jsTokens/colors.tokens.js

```
@@ -1,438 +1,438 @@
+    "400": "var(--rad-ui-color-orange-400)",
+    "500": "var(--rad-ui-color-orange-500)",
+    "600": "var(--rad-ui-color-orange-600)",
+    "700": "var(--rad-ui-color-orange-700)",
+    "800": "var(--rad-ui-color-orange-800)",
+    "900": "var(--rad-ui-color-orange-900)",
+    "950": "var(--rad-ui-color-orange-950)",
+    "1000": "var(--rad-ui-color-orange-1000)"
+  },
+  "black": "var(--rad-ui-color-black)",
+  "white": "var(--rad-ui-color-white)"
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)